### PR TITLE
chore(deps): move most dependencies to blsq-environment image

### PR DESCRIPTION
**Pull request by cheikhgwane**

### CODE REVIEW

File: images/blsq/Dockerfile [hunk 1]
```diff
@@ -21,10 +21,9 @@ RUN conda install --yes -c R \
   conda clean --all -y
 
 
-USER ${NB_UID}
```
Comment (cheikhgwane) [inline images/blsq/Dockerfile:24]: A package (rgdal ?) needs to update the gdal version installed and fails due to a permission denied.

File: images/base/Dockerfile [hunk 2]
```diff
@@ -56,39 +56,7 @@ ENV CONDA_PKGS_DIRS=/opt/conda/pkgs
...
-  'r-rcolorbrewer=1.*' \
-  'r-rjson=0.2.*' \
-  'r-rpostgres=1.*' \
-  'r-styler=1.*' \
-  'r-tidyverse=2.*' \
-  'r-viridis=0.6.*' \
-  'r-nloptr=2.*' \
-  # Python packages on conda-forge
-  'black=23.12.*' \
-  'fsspec=2023.12.2' \
```
Comment (qgerome) [inline images/base/Dockerfile:74]: This is needed everywhere

File: images/base/Dockerfile [hunk 3]
```diff
@@ -56,39 +56,7 @@ ENV CONDA_PKGS_DIRS=/opt/conda/pkgs
...
-  'gcsfs=2023.12.2' \
```
Comment (qgerome) [inline images/base/Dockerfile:75]: This is needed everywhere

File: images/base/Dockerfile [hunk 4]
```diff
@@ -56,39 +56,7 @@ ENV CONDA_PKGS_DIRS=/opt/conda/pkgs
...
-  'ipywidgets=8.*' \
-  'isort=5.*' \
-  'lxml=5.*' \
-  'nbresuse=0.4.*' \
-  'netCDF4=1.*' \
-  'papermill=2.*' \
```
Comment (qgerome) [inline images/base/Dockerfile:81]: This is needed everywhere

**Review by qgerome — CHANGES_REQUESTED** [review]
I also think the awscli can be moved to the legacy image

**Comment by cheikhgwane** [comment]
@qgerome all tested 👍

**Pull request merged**